ext/opentelemetry-ext-grpc/src/opentelemetry/ext/grpc/_client.py

```diff
@@ -72,28 +73,42 @@ def callback(response_future):
                 return
             response = response_future.result()
             rpc_info.response = response
+            if "ByteSize" in dir(response):
+                metrics_recorder.record_bytes_in(
+                    response.ByteSize(), client_info.full_method
+                )
 
     return callback
 
 
 class OpenTelemetryClientInterceptor(
     grpcext.UnaryClientInterceptor, grpcext.StreamClientInterceptor
 ):
-    def __init__(self, tracer):
+    def __init__(self, tracer, exporter, interval):
         self._tracer = tracer
+        self._meter = metrics.get_meter(__name__)
+        self.controller = PushController(
+            meter=self._meter, exporter=exporter, interval=interval
```

Comment:
`exporter` and `interval` could still be `None`?

Reply:
good catch - broke this when I rebased. Added a test to make sure things work without passing exporter

Reply:
I would be explicit about checking `interval` as well.

Reply:
done